src/Playwright/Helpers/Driver.cs

```diff
@@ -63,46 +63,68 @@ internal static string GetExecutablePath()
             assemblyDirectory = new FileInfo(assemblyLocation).Directory;
         }
 
-        string executableFile = GetPath(assemblyDirectory.FullName);
+        string executableFile;
+        Func<string, string> getArgs;
+
+        var driverSearchPath = Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_SEARCH_PATH");
+        if (!string.IsNullOrEmpty(driverSearchPath))
+        {
+            (executableFile, getArgs) = GetPath(Path.GetFullPath(driverSearchPath));
+            if (File.Exists(executableFile))
+            {
+                return (executableFile, getArgs);
+            }
+        }
+
+        (executableFile, getArgs) = GetPath(assemblyDirectory.FullName);
         if (File.Exists(executableFile))
         {
-            return executableFile;
+            return (executableFile, getArgs);
         }
 
         // if the above fails, we can assume we're in the nuget registry
-        executableFile = GetPath(assemblyDirectory.Parent.Parent.FullName);
+        (executableFile, getArgs) = GetPath(assemblyDirectory.Parent.Parent.FullName);
+        Console.Error.WriteLine(executableFile);
         if (File.Exists(executableFile))
         {
-            return executableFile;
+            return (executableFile, getArgs);
         }
 
         throw new PlaywrightException($"Driver not found: {executableFile}");
     }
 
-    private static string GetPath(string driversPath)
+    private static (string ExecutablePath, Func<string, string> GetArgs) GetPath(string driversPath)
     {
         string platformId;
-        string runnerName;
+        string nodeExecutable;
         if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
         {
             platformId = "win32_x64";
-            runnerName = "playwright.cmd";
+            nodeExecutable = "node.exe";
         }
         else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
         {
-            runnerName = "playwright.sh";
+            nodeExecutable = "node";
             platformId = RuntimeInformation.ProcessArchitecture == Architecture.Arm64 ? "darwin-arm64" : "darwin-x64";
         }
         else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
         {
-            runnerName = "playwright.sh";
+            nodeExecutable = "node";
             platformId = RuntimeInformation.ProcessArchitecture == Architecture.Arm64 ? "linux-arm64" : "linux-x64";
         }
         else
         {
             throw new PlaywrightException("Unknown platform");
         }
 
-        return Path.Combine(driversPath, ".playwright", "node", platformId, runnerName);
+        string getArgs(string args)
+        {
+            var cliEntrypoint = Path.Combine(driversPath, ".playwright", "package", "lib/cli/cli.js");
```

Comment:
Combine everything?
```suggestion
            var cliEntrypoint = Path.Combine(driversPath, ".playwright", "package", "lib", "cli", "cli.js");
```

Nit: calculate this once upfront outside of `getArgs`.